packages/react/src/tooltip/root/TooltipRoot.tsx

```diff
@@ -13,77 +13,43 @@ import { PortalContext } from '../../portal/PortalContext';
  * Documentation: [Base UI Tooltip](https://base-ui.com/react/components/tooltip)
  */
 const TooltipRoot: React.FC<TooltipRoot.Props> = function TooltipRoot(props) {
-  const { delay, closeDelay, hoverable = true, trackCursorAxis = 'none' } = props;
+  const {
+    defaultOpen = false,
+    onOpenChange,
+    open,
+    delay,
+    closeDelay,
+    hoverable = true,
+    trackCursorAxis = 'none',
+  } = props;
 
   const delayWithDefault = delay ?? OPEN_DELAY;
   const closeDelayWithDefault = closeDelay ?? 0;
 
-  const {
+  const tooltipRoot = useTooltipRoot({
+    ...props,
+    defaultOpen,
```

Comment:
What's the point of passing `onOpenChange`, `open`, and other props explicitly if they are a part of `...props`?

Reply:
Yeah, you're right. Forgot that the linting allows unused variables (as they need to be destructured to be documented)

Reply:
Well not anymore with our latest linting config:

> /tmp/base-ui/packages/react/src/popover/root/PopoverRoot.tsx
  16:11  error  'defaultOpen' is assigned a value but never used   @typescript-eslint/no-unused-vars
  16:24  error  'open' is assigned a value but never used          @typescript-eslint/no-unused-vars
  16:30  error  'onOpenChange' is assigned a value but never used  @typescript-eslint/no-unused-vars
...
✖ 12 problems (12 errors, 0 warnings)